src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -65,6 +65,9 @@ protected void onCreate(Bundle savedInstanceState) {
                     throw new IllegalArgumentException(
                             "You must provide the requested fragment type as an extra.");
             }
+        } else {
+            setTitle(R.string.settings_title);
+            mFragment = new SettingsFragment();
```

Comment:
What are the changes in this file about? They're definitely not needed for the LogActivity.